risc0/zkvm/src/host/client/prove/mod.rs

```diff
@@ -199,3 +199,29 @@ pub(crate) fn get_r0vm_path() -> PathBuf {
         .unwrap_or("r0vm".to_string())
         .into()
 }
+
+/// TODO: Write documentation.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[cfg_attr(test, derive(PartialEq))]
```

Comment:
Why not make this always have a `PartialEq`?

Reply:
Oh, I was following the pattern on what was defined on other structs, but can totally make it always have it!